File: E-TabanÇevirme/TabanCevirme.java
import java.util.*;
public class TabanCevirme {
 public static void main(String arg[]) {
  Scanner input = new Scanner(System.in);
  
  // Değişkenlerin tanımlanması
  int sayi, s = 0, i = 0, basamak;

  // Sayının girilmesi 
  sayi = input.nextInt();

  while (sayi > 0)
  {

   // Çevirme işleminin yapılması   
   basamak = (int)((sayi % 2) * Math.pow(10, i));
   i++;
   sayi = sayi / 2;
   s = s + basamak;
  }

  // Çevrilen sayının ekrana yazdırılması
  System.out.println(s); 
  input.close();
 }
}

File: E-TabanÇevirme/TabanCevirme.py
from math import pow
# Değişkenlerin tanımlanması
s = 0
i = 0
basamak = 0

# Sayının girilmesi
sayi = int(input())

while (sayi > 0):
    #  Çevirme işleminin yapılması
    basamak = int((sayi % 2) * pow(10, i))
    i += 1
    sayi = sayi // 2
    s = s + basamak

# Çevrilen sayının ekrana yazdırılması
print(s)
